yogstation/code/game/gamemodes/battle_royale/battleroyale.dm

```diff
@@ -64,9 +69,14 @@ GLOBAL_LIST_EMPTY(battleroyale_players) //reduce iteration cost
 	var/list/royalers = list()
 	if(GLOB.player_list.len <= 1) //It's a localhost testing
 		return
+	if(!GLOB.battleroyale_players.len) //sanity check for if this gets called before people are added to the list somehow
```

Comment:
use length(x) proc

Reply:
lazylen or length?

Reply:
also, should i change the other .len uses?